microsoft-edge/index.yml

```diff
@@ -0,0 +1,445 @@
+# -  Learn about Microsoft Edge in the enterprise
+# - -  Microsoft Edge channel overview - /deployedge/microsoft-edge-channels.md
+# - -  Download a Microsoft Edge channel - https://www.microsoftedgeinsider.com/download
+# - -  Use enterprise features - /deployedge/edge-ie-mode.md
+# -  Deploy and update Microsoft Edge
+# - -  Deploy to Windows - /configmgr/apps/deploy-use/deploy-edge.md
+# - -  Deploy to macOS - /intune/apps/apps-edge-macos/index.md
+# -  Configure Microsoft Edge
+# - -  Configure Microsoft Edge on Windows and macOS - /deployedge/configure-microsoft-edge.md
+# - -  Set Microsoft Edge as the default browser - /deployedge/edge-default-browser.md
+# -  Manage security and privacy
+# - -  Security - /deployedge/security-overview.md
+# -  Policy reference
+# - -  Browser policy reference - /deployedge/microsoft-edge-policies.md
+# - -  Update policy reference - /deployedge/microsoft-edge-update-policies.md
+# -  Engage with enterprise community 
+# - -  Enterprise tech community (Insider Focused) - https://techcommunity.microsoft.com/t5/Enterprise/bd-p/EdgeInsiderEnterprise
+# - -  Troubleshoot Microsoft Edge - /deployedge/microsoft-edge-troubleshooting.md
+
+# conceptualContent section (optional)
+conceptualContent:
+# itemType: architecture | concept | deploy | download | 
+#           get-started | how-to-guide | learn | overview | 
+#           quickstart | reference | tutorial | video | 
+#           whats-new
+  title: Reference # < 60 chars (optional)
+  summary: Cross-platform Web Apps, DevTools, and Extensions # < 160 chars (optional)
+  items:
+    # Card
+    - title: Progressive Web Apps
+      links:
+        - url: ./progressive-web-apps.md
+          itemType: reference
+          text: Overview
+        - url: ./progressive-web-apps/get-started.md
+          itemType: reference
+          text: Getting Started with PWAs
+        - url: ./progressive-web-apps/windows-features.md
+          itemType: reference
+          text: Tailor Your PWAs for Windows
+        - url: ./progressive-web-apps/microsoft-store.md
+          itemType: reference
+          text: PWAs in the Microsoft Store
+        - url: ./devtools-guide-chromium/progressive-web-apps.md
+          itemType: reference
+          text: Using DevTools with Progressive Web Apps
+    # Card
+    - title: WebView2
+      links:
+        - url: ./hosting/webview2.md
+          itemType: reference
+          text: Overview
+        - url: ./hosting/webview2/gettingstarted.md
+          itemType: reference
+          text: Getting Started with WebView2
+        - url: ./hosting/webview2/reference-webview2.md
+          itemType: reference
+          text: Reference
+    # Card
+    - title: Getting Started with DevTools
+      links:
+        - url: ./devtools-guide-chromium.md
+          itemType: reference
+          text: DevTools Overview
+        - url: ./devtools-guide-chromium/whats-new.md
```

Comment:
This isn't correct any more. I've actually removed this doc from the toc.md file. The currently correct path would be: `./devtools-guide-chromium/whats-new/2019/12/devtools.md` and we'll just have to update this for each What's New release we do.

Reply:
Updated.